.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./angular
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    # - name: Setup Chrome
+    #   id: setup-chrome
+    #   uses: browser-actions/setup-chrome@v1
+    #   with:
+    #     install-dependencies: true
```

Comment:
don't keep commented code in the pipeline - it makes reading much harder

Reply:
Actually I wanted to ask about necessity to include such step in the workflow. But your advise on how to run Karma tests helped to resolve my issue